test/unittests/test_stop_service.py

```
@@ -0,0 +1,556 @@
+
+        with patch.object(StopService, "get_active_skills", return_value=["bad_skill"]), \
+             patch("ovos_core.intent_services.stop_service.SessionManager.get",
+                   return_value=sess):
+
+            import threading
+            result_holder = []
+
+            def run():
+                try:
+                    result_holder.append(svc._collect_stop_skills(Message("test")))
+                except Exception:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)